### DISCUSSION TITLE
gherkinsible, a tiny example of Suitable usage

### DISCUSSION THREAD

**Discussion by csebille** [seantis/suitable#54] [Show and tell]
Hi,

  Thanks for publishing **suitable**

  Here is a little project I just started. It creates a bridge between behave and ansible thanks to suitable.
 
  https://github.com/csebille/gherkinsible

  The aim is to create more understandable tests leveraging the power of ansible and simplified by **suitable** of course :)

Best regards,
Christophe

**Comment by href** [comment]
Looks nice, thanks for sharing! Glad you found my project *suitable* for your project 😬
